easybuild/easyconfigs/c/Circos/Circos-0.69-6-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
```

Comment:
@SethosII Please restore the attribution that was there in the most recent `Circos` easyconfig (Francis Crick).